config/locales/find.yml

```diff
@@ -441,6 +446,14 @@ en:
           fee: Fee - no salary
           salary: Salary
           apprenticeship: Teaching apprenticeship - with salary
+        ordering:
+          location: Sorted by distance
```

Comment:
This is not a label ALTHOUGH I'd though it would make sense to everything about ordering to be here.

We don't display labeling when sorting by distance